app/scripts/modules/core/src/presentation/refresher/Refresher.tsx

```diff
@@ -39,6 +39,13 @@ export class Refresher extends React.Component<IRefresherProps, IRefresherState>
     });
   }
 
+  public componentWillReceiveProps(nextProps: IRefresherProps): void {
+    if (nextProps.refreshing !== this.props.refreshing ||
+        nextProps.lastRefresh !== this.props.lastRefresh) {
+      this.setState(this.parseState(nextProps));
+    }
+  }
```

Comment:
Already there in lines 36-39 👍 
I forgive you since it was probably in the rollup of code that wasn't changed.